metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -63,11 +63,13 @@ class Compilers(
     scribe.info(
       "no build target: using presentation compiler with only scala-library"
     )
-    val compiler = configure(new ScalaPresentationCompiler()).newInstance(
-      s"metals-default-${mtags.BuildInfo.scalaCompilerVersion}",
-      PackageIndex.scalaLibrary.asJava,
-      Nil.asJava
-    )
+    val ramboSearch = new RamboSymbolSearch(workspace)
```

Comment:
This constructor can fail now, which is not great since the rambo compiler is a lazy val. We should fallback to an empty symbol search on exception.